### PR TITLE
INTLY-3821 application monitoring operator rbac fix

### PR DESCRIPTION
**Pull request by davidkirwan**
Jira: https://issues.jboss.org/browse/INTLY-3821

Related PR: https://github.com/integr8ly/application-monitoring-operator/pull/86

Added the Application Monitoring Operator roles and service accounts to the integreatly-monitoring CSV

### CODE REVIEW

**Comment by mikenairn** [comment]
@davidkirwan Getting this error 

```
time="2019-10-10T17:14:40Z" level=info msg="Error: failed to create/update applicationmonitoring custom resource: applicationmonitorings.applicationmonitoring.integreatly.org is forbidden: User \"system:serviceaccount:integreatly:integreatly-operator\" cannot create resource \"applicationmonitorings\" in API group \"applicationmonitoring.integreatly.org\" in the namespace \"integreatly-middleware-monitoring\""
```

Is this what you were trying to fix?

**Comment by davidkirwan** [comment]
@mikenairn not exactly :laughing: but I think I can fix this by giving the integreatly operator the permissions to create this CR.

**Comment by davidkirwan** [comment]
@mikenairn I've pushed a fix, can you please retest once more.

File: integreatly/integreatly-1.9.4/integreatly-operator.1.9.4.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,407 @@
...
+    createdAt: 2019-10-09 10:39:00
+    support: Integreatly
+    description: Greatly integrating integreatly greatness
+spec:
+  displayName: Integreatly
+  description: Integreatly installation and reconciliation operator
+  keywords: ['integreatly']
+  version: 1.9.4
+  maturity: alpha
+  replaces: integreatlyoperator.1.9.3
```
Comment (mikenairn) [inline integreatly/integreatly-1.9.4/integreatly-operator.1.9.4.clusterserviceversion.yaml]: Is this going to be an issue if we're skipping 1.9.3?

Comment (philbrookes) [reply integreatly/integreatly-1.9.4/integreatly-operator.1.9.4.clusterserviceversion.yaml]: yes, this needs to be changed to 1.9.2 @davidkirwan

Comment (davidkirwan) [reply integreatly/integreatly-1.9.4/integreatly-operator.1.9.4.clusterserviceversion.yaml]: I think you're right, needs to be 1.9.2.

**Pull request merged**